app/models/upload.rb

```diff
@@ -44,6 +44,7 @@ class Upload < ApplicationRecord
     )
   }
 
+  validates :upload_order, absence: true, if: :owner_id?
```

Comment:
Hmm, isn't this backwards?  Shouldn't we be requiring the order if there's an owner, not the opposite?